nservicebus/upgrades/7to8/transport.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Transport configuration changes
+summary: Transport configuration changes from NServiceBus 7 to 8.
+reviewed: 2021-02-12
+component: Core
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ - 8
+---
+
+NServiceBus V8 comes with a new transport configuration API. Instead of the generic based `UseTransport<TTransport>` method, create an instance of the transport's configuration class and pass it to `UseTransport`, e.g.
+
+```csharp
+var transport = endpointConfiguration.UseTransport<MyTransport>();
+transport.Transactions(TransportTransactionMode.ReceiveOnly);
+var routing = t.Routing();
+routing.RouteToEndpoint(typeof(MyMessage), "DestinationEndpoint");
+```
+
+becomes:
+
+```csharp
+var transport = new MyTransport{
+    TransportTransactionMode = TransportTransactionMode.ReceiveOnly
+};
+var routing = endpointConfiguration.UseTransport(transport);
+routing.RouteToEndpoint(typeof(MyMessage), "DestinationEndpoint");
+```
+
+## Transport Specific Configuration
+
+See the transport specific upgrade guides for further details on the configuration options:
+
+* [Azure Service Bus transport upgrade guide](/transports/upgrades/asbs-1to2.md)
+* [Azure Storage Queues transport upgrade guide](/transports/upgrades/asq-9to10.md)
+* [RabbitMQ transport upgrade guide](/transports/upgrades/rabbitmq-6to7.md)
+* [MSMQ transport upgrade guide](/transports/upgrades/msmq-1to2.md)
+* [SQL Server transport upgrade guide](TODO)
```

Comment:
can we merge https://github.com/Particular/docs.particular.net/pull/5193 so we can go ahead with this?